docs/.vuepress/components/Button.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <!-- <h1>Test</h1> -->
+  <router-link v-if="isRouter" :to="to" :class="buttonClasses">
+    <slot />
+  </router-link>
+  <a href v-else-if="isA" :class="buttonClasses" :target="target" :rel="rel">
+    <slot />
+  </a>
+</template>
+
+<script>
+export default {
+  name: 'Button',
+  props: {
+    isA: {
+      type: Boolean,
+      default: true
+    },
+    isRouter: {
+      type: Boolean,
+      default: false
+    },
+    to: {
+      type: String
+    },
+    newTab: {
+      type: Boolean,
+      default: false
+    },
+    primary: {
```

Comment:
Same here - this boolean prop doesn't need to exist. The button can just default to primary unless `secondary` is passed in.